### ISSUE TITLE
Transformer left unconnected in schematics

### ISSUE THREAD

**Issue by Salamandar** [KiCad/kicad-symbols#1809]
I'm trying to use transformers, but the schematics looks completely lost. 
You can see on the screenshot that all components that **should** be connected to the transformer have an error "unconnected pin".
Obviously, the pins are still unconnected on the PCB.
I tried with different transformer symbols, same issue. That's the first time i get this kind of problem.

![Capture d’écran du 2019-05-02 13-36-51](https://user-images.githubusercontent.com/6552989/57072831-5ae56280-6cdf-11e9-81f9-c4aeea06331b.png)

**Comment by evanshultz** [comment]
@Salamandar 
What symbol are you using? The official library doesn't have a symbol named `Transformer_1P_1S`.

If this a symbol you made yourself, you really aren't in the right place to ask about this. This repo is regarding the official KiCad library. I would suggest https://forum.kicad.info/ for "using KiCad" questions and issues.

But anyway, my first thought is to check that the pins are on a coarse grid. Otherwise it can be hard to connect wires to pins. Our library uses a 100mil pin grid for this reason.

**Comment by Salamandar** [comment]
I'm using the Device:Transformer_1P_1S, in the official libraries.w
Found it here in the repo : https://github.com/KiCad/kicad-symbols/blob/844bc98d2a6f57f80e920b28181ab4bf68b5b39f/Device.lib#L11376

But, actually, I found that the bug is not in the libraries but in Kicad itself. When updating (sch -> PCB), the new components are not connected to the already-existing on the pcb.
I'll report it to the kicad issues.
